istio-application/templates/virtualservice.yaml

```
@@ -33,10 +33,12 @@ metadata:
   name: {{ .Values.global.name }}
 spec:
   hosts:
-    - "{{ .Values.global.host }}.{{.Values.global.dnsDomain}}"
+    - {{ .Values.global.host }}.{{.Values.global.dnsDomain}}
```

Please be cosistent with the templating.
```suggestion
    - {{ .Values.global.host }}.{{ .Values.global.dnsDomain }}
```